### PR TITLE
lab: add new commands "db"

### PR DESCRIPTION
**Pull request by falzm**
This change introduces beta support for DBaaS management commands under
the `exo lab db` section.

### CODE REVIEW

File: cmd/deploy_target_list.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (c *deployTargetListCmd) cmdLong() string {
 	return fmt.Sprintf(`This command lists existing Deploy Targets.
 
 	Supported output template annotations: %s`,
-		strings.Join(outputterTemplateAnnotations(&vmListOutput{}), ", "))
+		strings.Join(outputterTemplateAnnotations(&deployTargetListOutput{}), ", "))
```
Comment (zyegfryed) [inline cmd/deploy_target_list.go:39]: Nice catch ;)

File: cmd/lab.go [hunk 2]
```diff
@@ -8,7 +8,11 @@ var labCmd = &cobra.Command{
 	Use:   "lab",
 	Short: "Experimental commands",
 	Long: `These commands provide work-in-progress functionalities that may or
-may not be promoted to production someday. Caution: wet paint.`,
+may not be promoted to production someday.
+
+/!\ IMPORTANT: Exoscale provides no guarantees regarding the stability of the
+commands provided in this section, and their syntax can change without prior
+notice.`,
```
Comment (zyegfryed) [inline cmd/lab.go:15]: 👍🏻

File: cmd/vm_start.go [hunk 3]
```diff
@@ -50,23 +50,6 @@ var vmStartCmd = &cobra.Command{
...
-	state := (string)(egoscale.VirtualMachineStopped)
-	if vm.State != state {
-		return fmt.Errorf("%q is not in a %s state, got: %s", vmName, state, vm.State)
-	}
-
-	_, err = asyncRequest(&egoscale.StartVirtualMachine{ID: vm.ID, RescueProfile: vmRescueProfile},
-		fmt.Sprintf("Starting %q ", vm.Name))
-	return err
-}
-
```
Comment (zyegfryed) [inline cmd/vm_start.go:69]: Dead code, right?

File: cmd/vm_stop.go [hunk 4]
```diff
@@ -56,22 +56,6 @@ var vmStopCmd = &cobra.Command{
...
-
-	state := (string)(egoscale.VirtualMachineRunning)
-	if vm.State != state {
-		return fmt.Errorf("%q is not in a %s state, got %s", vmName, state, vm.State)
-	}
-
-	_, err = asyncRequest(&egoscale.StopVirtualMachine{ID: vm.ID}, fmt.Sprintf("Stopping %q ", vm.Name))
-	return err
-}
-
```
Comment (zyegfryed) [inline cmd/vm_stop.go:74]: Dead code, right?

File: cmd/vm.go [hunk 5]
```diff
@@ -88,18 +88,6 @@ func saveKeyPair(keyPairs *egoscale.SSHKeyPair, vmID egoscale.UUID) {
...
-
-	if _, err := os.Stat(folder); !os.IsNotExist(err) {
-		if err := os.RemoveAll(folder); err != nil {
-			return fmt.Errorf("the SSH private key could not be deleted: %s", err)
-		}
-	}
-
-	return nil
-}
-
```
Comment (zyegfryed) [inline cmd/vm.go:102]: Dead code, right?

**Review by zyegfryed — APPROVED** [review]
🚀 
**PS:** AFAIU we also remove all the `lab kube` command at the same time, right?

File: cmd/vm_start.go (hunk 3 above)
Comment (falzm) [reply cmd/vm_start.go:69]: Yes, used only by the `exo lab kube` commands.

File: cmd/vm_stop.go (hunk 4 above)
Comment (falzm) [reply cmd/vm_stop.go:74]: Yes, used only by the `exo lab kube` commands.

File: cmd/vm.go (hunk 5 above)
Comment (falzm) [reply cmd/vm.go:102]: Yes, used only by the `exo lab kube` commands.

**Comment by falzm** [comment]
> 🚀
> **PS:** AFAIU we also remove all the `lab kube` commands at the same time, right?

Yes, those are being retired.

**Pull request merged**